### DISCUSSION TITLE
Run bundled browser frontend/backend

### DISCUSSION THREAD

**Discussion by Chris-Holter** [eclipse-theia/theia#13387] [Q&A] [answered]
Hi,
this might be a newbie question - so excuse me for that ;)

I understood that webpack bundles the backend and frontend of e.g. the theia blueprint ide in the lib folder under application/browser/

But I'm missing how to execute the frontend and backend if I'd like to distribute the bundled files, e.g. in a docker container. Can somebody point me in the right direction here? Or better: Give me an example?

**Comment by msujew** [comment, marked as answer]
Hey @Chris-Holter,

the frontend is being served by the backend, so it's only the backend you need to start. Once the backend is bundled, you should be able to just execute `node lib/backend/main.js` in the backend app directory to start the server.

**Comment by sdirix** [comment]
Hi @Chris-Holter,

You can find a Docker example in the official Theia Blueprint / IDE repository, see [here](https://github.com/eclipse-theia/theia-blueprint/blob/5e02586f0eb3add800a268f07e4a4c6b2215ff55/browser.Dockerfile).

**Comment by Chris-Holter** [comment]
Thanks a lot for the quick answers!
